tests/tests-misc.php

```diff
@@ -614,4 +614,24 @@ public function test_cart_url_formats() {
 
 		EDD_Helper_Download::delete_download( $post->ID );
 	}
+
+	public function test_array_convert() {
+		$customer1_id = EDD()->customers->add( array( 'email' => 'test10@example.com' ) );
```

Comment:
These tests should be expanded to check for various properties inside of the object. That will help prevent against this: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/4042#issuecomment-220081416